server/monitor/src/main/java/org/apache/accumulo/monitor/util/Table.java

```diff
@@ -143,7 +146,24 @@ public synchronized void generate(HttpServletRequest req, StringBuilder sb) {
       showLegend = showStr != null && Boolean.parseBoolean(showStr);
     }
 
-    sb.append("<div>\n");
+    String redir = BasicServlet.currentPage(req);
+
+    if (namespaces != null) {
+      sb.append("<div id=\"filters\">\n");
+      String namespace = BasicServlet.getCookieValue(req, "namespaceDropdown." + BasicServlet.encode(page) + "." + BasicServlet.encode(tableName) + "."
+          + "selected");
+      if (namespace == null) {
+        namespace = "*";
+      }
+      sb.append("<div class='left show'><dl>\n");
+      doDropdownMenu(redir, page, tableName, sb, namespaces, namespace);
```

Comment:
Will add a header "Namespaces" that uses a different font than the rest of the list.